From bdd282ca93e94c95667e30ae3beddf8ae85f4047 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 16 Jul 2019 16:44:20 -0700
Subject: [PATCH] TRITON-1818 explicitly specify PEM key format for keys
 generated by ssh-keygen Reviewed by: Angela Fong <angela.fong@joyent.com>
 Approved by: Angela Fong <angela.fong@joyent.com>

---
 docs/index.md                  | 2 +-
 test/volumes-automount.test.js | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/docs/index.md b/docs/index.md
index e9ed3b1..2fa389d 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -133,7 +133,7 @@ to many instances.
 If you haven't already generated an SSH key (required to use both SSH and HTTP
 Signing), run the following command:
 
-    $ ssh-keygen -b 2048 -t rsa
+    $ ssh-keygen -b 2048 -t rsa -m PEM
 
 This will prompt you with a place to save the key.  You should probably just
 accept the defaults, as many programs (SSH and CloudAPI CLIs) will first look
diff --git a/test/volumes-automount.test.js b/test/volumes-automount.test.js
index e47abea..d7d9562 100644
--- a/test/volumes-automount.test.js
+++ b/test/volumes-automount.test.js
@@ -215,6 +215,7 @@ if (CONFIG.experimental_cloudapi_nfs_shared_volumes !== true) {
         child_process.exec([
             'ssh-keygen',
             '-t rsa',
+            '-m PEM',
             '-N ""',
             '-f',
             KEY_FILENAME
-- 
2.21.0

